backend/src/db-utils.js

```diff
@@ -546,6 +546,77 @@ const queryBridgeTokenHolders = async (bridgeTokenContractId) => {
   );
 };
 
+const queryBridgeMintTransactionAction = async () => {
```

Comment:
The name implies you only need a single action, but implementation fetches all of them. Do you really need all of them? It seems to me it will grow indefinitely, so this query will cause problems.

We need to limit queries to something manageable.

Reply:
We really need all of them, but will not query that frequently

Reply:
@icerove BTW, would direct access to the database be more convenient for these use-cases? See, we now have one: https://github.com/near/near-indexer-for-explorer#shared-public-access

Reply:
direct access to the database not very make sense, I have to query first and then deal with my end

Reply:
you are right, I will remove these bridge and market query from explorer side and use direct access